EventsExpress/ClientApp/src/components/privacy/privacy.js

```diff
@@ -0,0 +1,69 @@
+        Once you delete your account, it will take some time to be completely removed from the system. However, other users will not be able to see your account, although they may be able to read your public postings and comments you have made.
+        </p>
+        <h4>Third-Party Accounts</h4>
+        <p>You will have an option to create an EventsExpress account through an email or through an already existing Facebook or Google account. EventsExpress does not store your EventsExpress, Facebook, or Google account passwords. Your account information is not disclosed to EventsExpress and you are free to cancel any of your social networking website connections with EventsExpress at any time. We do not collect or gain access to your Facebook or Google pictures, locations, statuses, or any other data unless it is made public, which could include such as email addresses, profile pictures, friend lists, and birthdates. You will have an option to choose if you want to invite your Facebook friends to EventsExpress and what information you share via Facebook or other networking sites. We do not control this and we are not responsible for the content in third-party accounts</p>
+        <h4>Third-party websites</h4>
+        <p>Our website may contain several places where you are able to click on a link to access other websites, such as for information and advertisement. These websites do not operate under this Privacy Policy and we do not control them or what information they might collect from you. You should check each third-party website's own Privacy Policies and Terms and Conditions. Other users may provide links as well, but we are not responsible for the content nor do we control or check them. You should access those websites at your own risk.</p>
+        <h4>Changes To This Privacy Policy</h4>
+        <p>This Privacy Policy is effective as of March 1st, 2022 and will remain in effect except with respect to any changes in its provisions in the future, which will be in effect immediately after being posted on this page. 
+        We reserve the right to update or change our Privacy Policy at any time and you should check this Privacy Policy periodically. Your continued use of the Service after we post any modifications to the Privacy Policy on this page will constitute your acknowledgment of the modifications and your consent to abide and be bound by the modified Privacy Policy. If you do not consent to such changes you should not continue to use the services.
+        </p>
+        <h4>Enforcement</h4>
+        <p>We regularly review our own compliance with our Privacy Policy. If you submit a formal written complaint with your contact information, we will try our best to resolve the issue.</p>
+        <h4>Contact Us</h4>
+        <p>If you have any questions about this Privacy Policy or notice activity against it, please contact us. 
+            
+        </p>
+
+        <p>address</p>
```

Comment:
Should there be some real address? If not - just remove it, or add e-mail of some sorts.